.github/workflows/_integration_tests.yml

```diff
@@ -97,22 +97,30 @@ jobs:
       fail-fast: false
       matrix:
         test: [
```

Comment:
If it's easy, alphabetize these? That way it's easy to look at this and the file listing and be sure that every script gets called.

Reply:
e.g. Github has `direct-curl-portal-relay-down` before `direct-curl-portal-restart` 
<img width="1254" alt="image" src="https://github.com/firezone/firezone/assets/13400041/af737e68-e2f7-40dc-bd64-e0d36ee1a31e">


Hopefully that is not locale-sensitive since it's all ASCII letters with no numbers 😨

Reply:
Alphabetized